src/fielddef.ts

```diff
@@ -20,6 +20,8 @@ export interface FieldRefOption {
   binSuffix?: string;
   /** append suffix to the field ref (general) */
   suffix?: string;
+  /** return the base field, not rank */
+  noRank?: boolean;
```

Comment:
I'm curious if we could simply do `prefn: 'rank_'`  for the case where we need rank 
or at least do `rank` rather than `noRank`.   That should easily solve the problem I mentioned earlier.